sdcm/mgmt/cli.py

```diff
@@ -806,8 +810,8 @@ def resume(self, start_tasks=True):
         self.sctool.run(cmd=cmd)
 
     @contextmanager
-    def suspend_manager_then_resume(self, start_tasks=True):
-        self.suspend()
+    def suspend_manager_then_resume(self, start_tasks=True, start_tasks_in_advance=False, duration=None):
```

Comment:
what is this duration about? is it the time it will be stopped? or is it some kind of delay until restarted?

Reply:
It's the time that if the cluster is resumed before it passes, the tasks will not be continued on resume

Reply:
perhaps a better naming for `duration` is required

Reply:
I understand why you'd like this change, but I'd rather keep the function arguments name the same as the name given to them in the cli